### ISSUE TITLE
Chore: write alt text for illustrations

### ISSUE THREAD

**Issue by cmajel** [cal-itp/mobility-marketplace#782]
Per slack thread, we need to write alt text for illustrations. 

Guidance includes: 
* https://www.section508.gov/create/alternative-text/
* https://webaim.org/techniques/alttext/
